_posts/2022-01-10-reassign-partitions.md

```diff
@@ -0,0 +1,435 @@
+---
+layout: post
+title:  "Reassigning partitions in Apache Kafka Cluster"
+date: 2021-09-23
+author: shubham_rawat
+---
+
+As Apache Kafka users, we sometimes have to scale up/down the number of Kafka brokers in our cluster depending on the use case.
```

Comment:
You might want to say something more concrete about the "use case".

e.g. 
"...we sometimes want to scale up/down the number of Kafka brokers in our cluster in response to the load on the cluster."

I guess scaling in response to load is the only reason? (Up for greater load, down when you can save money because the load is low.)